components/needs_team/needs_team.tsx

```diff
@@ -139,6 +139,11 @@ export default class NeedsTeam extends React.PureComponent<Props, State> {
 
     public componentDidMount() {
         startPeriodicStatusUpdates();
+
+        if (this.state.team) {
```

Comment:
It was added here https://github.com/mattermost/mattermost-webapp/pull/11135/files#r973003979 but reverted to solve the issue temporarily.